manifest_base.yml

```diff
@@ -14,4 +14,4 @@ applications:
     host: eregsnc-demo
   - name: eregs-worker
     no-route: true
-    command: celery worker --app notice_and_comment
+    command: celery worker --app notice_and_comment -l DEBUG
```

Comment:
I did try and add the DEBUG level logging in just the manifest_dev.yml, and that is what gave me all the ""failed to accept connections within health check timeout" errors (I suspect).